### PR TITLE
Add cluster route table and feature value for VPC peering

### PR DESCRIPTION
**Pull request by florianlepont**
- I added missing parameter "accepted_values" in **/{cloudProvider}/ClusterFeature** endpoint to get available values
- I created  **/organization/{organizationId}/cluster/{clusterId}/routingTable** endpoint to get and edit routing table 
- These updates meet the needs for VPC peering

### CODE REVIEW

**Comment by evoxmusic** [comment]
Do you validate it @erebe ? I am not sure the format is good enough (String) - is it what you expect?

File: src/resources/OrganizationClusterRoutingTable.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          schema:
+            $ref: '../schemas/ClusterRoutingTableResponse.yaml'
+    '401':
+      $ref: '../responses/NotAuthorized.yaml'
+    '403':
+      $ref: '../responses/Forbidden.yaml'
+    '404':
+      $ref: '../responses/NotFound.yaml'
+
+post:
```
Comment (erebe) [inline src/resources/OrganizationClusterRoutingTable.yaml:24]: put is more common for editing.
post is usually for creation

File: src/schemas/ClusterFeatureResponse.yaml [hunk 2]
```diff
@@ -32,3 +32,6 @@ properties:
   is_value_updatable:
     type: boolean
     default: false
+  accepted_values:
+    type: array
+    nullable: true
```
Comment (erebe) [inline src/schemas/ClusterFeatureResponse.yaml:37]: It is not nullable

File: src/schemas/ClusterRoutingTableResponse.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+type: object
+properties:
+  results:
+    type: array
+    items:
+      type: object
+      properties:
+        destination:
+          type: string
+          nullable: false
```
Comment (erebe) [inline src/schemas/ClusterRoutingTableResponse.yaml:10]: not nullable

Comment (florianlepont) [reply src/schemas/ClusterRoutingTableResponse.yaml:10]: I set the nullable parameter as false, do you want me to remove the parameter ? or are you talking about the description parameter?

**Review by erebe — APPROVED** [review]

**Pull request merged**